src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/range.tsx

```diff
@@ -19,19 +19,26 @@
 
 import React from 'react';
 import { EuiFormRow, EuiRange } from '@elastic/eui';
-import { VisOptionsSetValue } from 'ui/vis/editors/default';
 
-interface RangeOptionProps {
+interface RangeOptionProps<ParamName extends string> {
```

Comment:
Seems as duplicated in string 33

Reply:
Explained this offline, why typescript needs this duplicate code generic narrowing.